content/about/main/index.md

```diff
@@ -7,27 +7,27 @@ number_categories: 3 # set to zero to exclude
 show_intro: true
 intro: |
 
```

Comment:
I moved two paragraphs so that the structure could be something like

- "paid" career
- community stuff
- personal stuff

Reply:
Thanks!